README.md

```diff
@@ -3,6 +3,7 @@
 [![Docs status][docs-badge]][docs-link]
 [![PyPI version][pypi-badge]][pypi-link]
 [![License][license-badge]][license-link]
+[![DOI](https://zenodo.org/badge/750834002.svg)](https://zenodo.org/badge/latestdoi/750834002)
```

Comment:
Doesn't matter too much but potentially could be worth moving the badge/link so together with the others.